02-intro-to-genomic-data.Rmd

```diff
@@ -5,10 +5,67 @@ Borrow from https://hutchdatascience.org/Choosing_Genomics_Tools/
 
 
 ## What is genomic data?
+Genome: Complete set of DNA in an organism.
```

Comment:
```suggestion
 - `Genome`: Complete set of DNA in an organism.
```

Minor comment but for a glossary like this might be nice to have a bullet point list.

Reply:
I love the glossary! I think it might be better after the paragraph? What do you think?